qa/tasks/cephfs/test_nfs.py

```diff
@@ -10,6 +10,7 @@
 log = logging.getLogger(__name__)
 
 
+# TODO Add test for cluster update when ganesha can be deployed on multiple ports.
```

Comment:
Is this a limitation of the ceph-adm or issue with deploying them in teuthology environment? Is there a tracker ticket for this issue?

Reply:
Currently cephadm does not support it. @sebastian-philipp

Reply:
interesting! what do you need? setting the port when deploying a service?

Reply:
Deploying multiple ganesha daemons on a single host.